### ISSUE TITLE
Deep links 

### ISSUE THREAD

**Issue by ajsb85** [telegramdesktop/tdesktop#3297]
Every new deep link (ex: https://t.me/ajsb85) opens a new instance of Telegram Desktop.

I installed the app from the Windows Store.

![image](https://cloud.githubusercontent.com/assets/663460/25169451/65d37d18-24e7-11e7-9be3-60a87f68af25.png)

Version 1.0.29

**Comment by auchri** [comment]
@ajsb85 Use the template please.

**Comment by ajsb85** [comment]
@auchri Done

**Comment by john-preston** [comment]
@ajsb85 Does it continue to happen after system reboot?

**Comment by ajsb85** [comment]
@john-preston Yes. Always. I uninstall and install, reboot the system, closing the app,...  and happen.
